mobile-development/mobile-development.md

```diff
@@ -13,7 +13,20 @@ About 4 hours
 - [React Lesson](/react-js/react.md)
 
 ### Motivation
+[(by Sara Bhavsar)](https://www.quora.com/Why-is-the-mobile-application-development-important)
```

Comment:
No, that is not what I meant.  Remove this line.  The paragraph needs formatting or grammar fixes.  Please see my previous comment.